samples/ResourceGroup/copy-resourcegroup-tag/azurepolicy.json

```diff
@@ -0,0 +1,22 @@
+{
+    "properties": {
+        "displayName": "Copy resource group tag to resource",
+        "mode": "Indexed",
+        "description": "Copy a named tag from the resource group tag to the resource.  For example, a tag named example on the resource group will be copied to the resources.  Change the word example in the policy rule to change the name of the tag copied to a resource.",
+        "policyRule": {
+            "if": {
+                "field": "tags.example",
+                "notEquals": "[resourceGroup().tags.example]"
```

Comment:
I missed this earlier! This should just be 
"exists": "false"

------------

If we use "notEquals", the policy will attempt to update the resource's *example* tag if it has a different value from the resource group's *example* tag. This would be cool, except that the "append" effect can only add _new_ tags to a resource. If the tag already exists with a different value, the policy would reject the resource 🙁  Using "exists" saves us from this by only appending the tag if the tag doesn't exist yet.